app/assets/javascripts/controllers/alerts/edit_alert_dialog_controller.js

```diff
@@ -0,0 +1,6 @@
+angular.module('alertsCenter')
+  .controller('EditAlertDialogController',['$scope', 'vm',
+    function($scope, vm) {
+      $scope.vm = vm;
```

Comment:
Please, use `controllerAs` instad of `$scope` for new code. Also, please, don't inject `$scope` when not needed.